lib/Archive/Tar/Stream.pm

```diff
@@ -14,18 +14,19 @@ use File::Temp;
 
 # XXX - make this an OO attribute
 our $VERBOSE = 0;
+our $NODIE = 0;
```

Comment:
This should be an OO attribute on $Self.  I notice that you've set one of those as well, so we should use that throughout really.